source/playbook/index.html.erb

```diff
@@ -17,33 +17,42 @@ directory_index: false
       
       <h2 id="intended_audience">Who?</h2>
       <p>"Who" is the intended audience for the Playbook? The Playbook will be valuable to a range of organizations and people interested in creating and leveraging specialty health data standards to address real-world challenges.</p>
       <p>The Playbook should be particularly helpful for specialty leaders exploring how to replicate the CodeX model for oncology within their own specialty. Specialty leaders may be motivated to work with CodeX and other specialties to speed work and increase coherency.</p>
-<p>Parts of the Playbook will be useful to particular stakeholders (e.g., patient organizations, health systems, specialty societies, government agencies, payers, vendors, etc.) and roles (e.g., decision makers, funders, project managers, community organizers, clinical experts, informaticists, standards developers, system engineers, communications personnel, etc.) involved in just a subset of the spectrum of all future work. The <%= link_to("Key Playbook Sections by Role", "/resources/roles.html") %> provided here aims to help guide people with leadership, clinical, or technical responsibilities to quickly find the most relevant information for them.</p>
-<p>
-To get the most out of the Playbook, it would be helpful for leaders to have at least a high-level knowledge of standards that are foundational in the Playbook (e.g., the <a href="https://www.healthit.gov/isa/united-states-core-data-interoperability-uscdi">United States Core Data for Interoperability (USCDI)</a>, <a href="https://hl7.org/fhir/summary.html">Fast Healthcare Interoperability Resources (FHIR)</a>, <a href="https://www.hl7.org/fhir/us/core/">US Core FHIR Implementation Guide</a>, etc.).  Standards implementers should, of course, have a much deeper understanding of these foundations. </p>
+      <p>Parts of the Playbook will be useful to particular stakeholders (e.g., patient organizations, health systems, specialty societies, government agencies, payers, vendors, etc.) and roles (e.g., decision makers, funders, project managers, community organizers, clinical experts, informaticists, standards developers, system engineers, communications personnel, etc.) involved in just a subset of the spectrum of all future work. The <%= link_to("Key Playbook Sections by Role", "/resources/roles.html") %> provided here aims to help guide people with leadership, clinical, or technical responsibilities to quickly find the most relevant information for them.</p>
+      <p>
+      To get the most out of the Playbook, it would be helpful for leaders to have at least a high-level knowledge of standards that are foundational in the Playbook:
```

Comment:
Make this more direct. "It is recommended that community members have a high-level knowledge of the foundational standards involved in use case development, including:"
It is shorter, and avoids directly pointing out that anyone, including leadership may not know standards.